### DISCUSSION TITLE
Have Collection Groups

### DISCUSSION THREAD

**Discussion by mikumin27** [ppy/osu#24676] [Ideas] [answered]
Id like a way to put collection in something like a folder.

Initially I and a friend group thought of this because we are planning to advertise the game in a convention and we wanted to make collections for gamemodes and for genres. In the end we came to the Idea that it would make sense to have collections named by genres in a group that is named by the gamemode so that its not too overwhelming for newbies.

Even outside of this specific scenario I think this could be useful. For example you could start making Collection groups for farm maps and practice maps and maybe for favourites which you later specify the difficulty or Maptype like streams or jumps in the collection itself.

In the end it is just to make big amounts of Collections easier to menu.

**Comment by monochrome22** [comment, marked as answer]
This idea was talked about here: https://github.com/ppy/osu/discussions/15397.

**Comment by soopax** [comment]
Hello,
> In the end we came to the Idea that it would make sense to have collections named by genres in a group that is named by the gamemode so that its not too overwhelming for newbies.

In lazer, you could simply add the beatmaps from all gamemodes into the same collection. You can pick the gamemode from the top left in the toolbar. When switching to a collection from the dropdown in the song selection, it only shows beatmaps that are in the collection and from the selected gamemode. Therefore you wouldn't need to create separate "folders" for different gamemodes.

**Reply by mikumin27** [reply]
Hi, thank you for this information. Initially I thought it would be a bad idea since it was converting standard maps into other gamemodes when changed but a specific setting was able to turn that off. We will be doing this with that feature. Thank you again!
